<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RSS reader</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0" colspan="4">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="4">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLineEdit" name="txtLink"/>
     </item>
     <item row="7" column="3" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btnView">
       <property name="text">
        <string>View feed</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="lblAdd">
       <property name="text">
        <string>RSS link:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="txtName"/>
     </item>
     <item row="7" column="1">
      <widget class="QRadioButton" name="rbText">
       <property name="text">
        <string>Text</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="lblView">
       <property name="text">
        <string>View in:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>Delete feed</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QRadioButton" name="rbHtml">
       <property name="text">
        <string>HTML</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btnImport">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Add feed</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QListWidget" name="listFeeds">
       <property name="editTriggers">
        <set>QAbstractItemView::CurrentChanged|QAbstractItemView::SelectedClicked</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
